src/USDA.NET/USDA.NET.csproj

```diff
@@ -1,14 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageId>USDA.NET</PackageId>
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <PackageId>USDA.API</PackageId>
```

Comment:
I don't think changing the PackageId makes sense here. The GitHub project is named USDA.NET,  there is already a published NuGet package that is being consumed, and `.NET` is a popular extension to add to .NET clients.

Reply:
I changed the package name and version because I needed this is in a project I'm working on and I wasn't sure you were paying attention to this repo any more. I'm happy to change it back and let you republish with whatever version you see fit.

Reply:
@Eonasdan : Yes, please put back the package name and version. After that, I can merge all the changes, bump the package version by 1 and then update the NuGet package. These changes are great and I would definitely like to merge them. Thank you for your contribution. :)

Reply:
Done. If the the version isn't to your liking, please change it yourself or tell me what you want the csproj package properties page to look like